genEnvVarFile.py

```diff
@@ -80,7 +80,7 @@ def genEnvs():
     # * --------------
     vars["LMDFIT_MACROPATH"] = ask(
         "LMDFIT_MACROPATH",
-        f"{vars['VMCWORKDIR']}/macro/detectors/lmd",
```

Comment:
This is just a default suggestion, the user is asked to change if needed. let's revert to the original